guides/user/managed-delivery/faq/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: single
+title:  "FAQ"
+sidebar:
+  nav: guides
+redirect_from: /reference/managed-delivery/
+---
+
+{% include toc %}
+
+This is a quick reference for Managed Delivery. 
+
+### Where am I?
+
+You probably arrived here from a tooltip on a declaratively managed resource.
+This page has some helpful information to get oriented.
+
+
+### What is happening?
+
+Spinnaker is managing this resource. 
+That means that the state of this resource is controlled using a declarative config submitted to Spinnaker.
```

Comment:
Will "this resource" be clear to all/most who are on this page? IOW, I think it's possible I might arrive on this FAQ from browsing the docs and there would be no contextual resource.

Reply:
Agreed, but since this is moved content, we'll rework or replace in a future PR.